Develop

stengun opened the pull request:
This pull request will add the new -o option to write hashes in the common used format (like HASH FILENAME) trying to satisfy the #4 enhancement request.
I removed -b and -j, options never used and bad implemented (because nhoya barf barf)
other small improvements regarding message printing (I think the program needs more testing)

Nhoya committed:
added dependencies check (uses -b, still in beta)

Commit:
Added dependencies check before execute

Nhoya committed:
removed  -b option, fixed errors in dependencies check, added _exit 1

stengun committed:
Italian word but english project

stengun committed:
Pre commit

stengun committed:
Format adjustments

stengun committed:
Add custom echo function to handle output to file.

stengun committed:
Small adaptations

stengun committed:
Add output to file, using standard format

stengun committed:
Remove unused option -b

Nhoya commented:
TODO: 
fix basename, solve the problem with special char on file path, verbose hr mode for -o option

Nhoya committed:
_writetofile() check for hash existence before printing on file

stengun committed:
Fix bad behavior with files named as parameters

stengun committed:
Merge branch 'develop' of github.com:Nhoya/gensum into develop

stengun committed:
Fix bad behavior with archives

stengun committed:
Update usage screen for github

stengun committed:
Update README.md

stengun committed:
Update README.md

intentation nazi

stengun committed:
Update README.md

More Indentation nazi

stengun committed:
Update README.md

Even more fucking indentation nazi

stengun committed:
make directory file listing more reliable

stengun committed:
Merge branch 'develop' of github.com:Nhoya/gensum into develop

stengun committed:
hide dotfiles by default

stengun committed:
Add -H option to include dotfiles

stengun commented:
Right now I added new enhancements, like a more reliable descending method for directories, and a new option to include hidden files while descending dirs.

stengun committed:
resolving conflicts with README.md

stengun committed:
Add a simple progress bar (wip)

stengun committed:
Remove myself from criminal records

stengun committed:
Add options and better progress bar

stengun committed:
Remove unused commands

stengun committed:
Update README.md

Yes, I'm here again, but not for indentation nazi, is because I added a new -x option.

Commit:
fixed typos, fixed indentation for help
